src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#"
+          class="header__img"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
```

Comment:
The `alt` attribute for the logo image should describe the image content, not just say 'logo'. For example, if this is a Moyo header logo, a more descriptive `alt` text would be 'Moyo header logo'.